org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/comment-snippets.json

```diff
@@ -7,6 +7,7 @@
     "body": [
       "<!-- $1-->"
     ],
-    "description": "Insert Comment"
+    "description": "Insert Comment",
+    "sortText": "ZZa"

```

Comment:
sinze 'a'(== 97) > `A`(==65), should the sortext be `zzA` instaed to not make it lower than eg `Hello` ?

Reply:
Your usecase Hello works with my current PR:

![image](https://user-images.githubusercontent.com/1932211/149560237-1e598fa7-1aed-43cf-ad2b-72c0d9ab4ab3.png)

Reply:
Maybe vscode does lozercase the sorttext or something. I'll try with LSP4E soon.